.travis.yml

```diff
@@ -3,17 +3,16 @@ os:
     - linux
     - osx
 julia:
-    - 0.4
-    - 0.5
-    - 0.6
+    - 0.7
+    - 1.0
```

Comment:
REQUIRE file and appveyor.yml also need to be updated

Reply:
Done